### PR TITLE
Added Innogy/RWE Charging stations

### PR DESCRIPTION
**Pull request by DavidMoraisFerreira**
Hi,

This PR adds the charging stations of Innogy/RWE, as per the [OSM Wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcharging_station#Examples).

### CODE REVIEW

**Comment by bhousel** [comment]
Looks good, thank you @DavidMoraisFerreira  👍

File: brands/amenity/charging_station.json [hunk 1]
```diff
@@ -90,7 +90,17 @@
...
+    "matchNames": ["RWE"],
+    "tags": {
+      "amenity": "charging_station",
+      "brand": "Innogy",
+      "brand:wikidata": "Q2124721",
+      "brand:wikipedia": "de:Innogy",
+      "name": "Innogy"
+    }
+  },
+  "amenity/charging_station|Source-London": {
```
Comment (tas50) [inline brands/amenity/charging_station.json]: @DavidMoraisFerreira did you mean to change the name of Source London as well?

Comment (DavidMoraisFerreira) [reply brands/amenity/charging_station.json]: Oh my! I did not intend to commit that change, sorry about that! I was trying to figure out why npm was throwing errors. Shall I open a new PR with the fix?

Comment (DavidMoraisFerreira) [reply brands/amenity/charging_station.json]: I pushed the changes to #3101.

**Pull request merged**